### PR TITLE
fix(#794) - Change footer links for and other fixes for angular base SPAs

### LINKED ISSUE #794: Change footer links for and other fixes for SPAs
## Is your chore related to a problem? Please describe 

Incorrect links in the footer

## Describe the solution you'd like

- Change footer links
  - Feedback
  - Home
  - Notifications
- Change notification spa with onlogin method

## Describe alternatives you've considered

None

## Additional context

N/A

### PR DESCRIPTION
**Pull request by hybridx**
### Resolves #794 

# Explain the feature/fix

- Changed footer JSON

## Does this PR introduce a breaking change

No

### Ready-for-merge Checklist

- [x] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [x] Does the change have appropriate unit tests?
- [x] Did tests pass?
- [x] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [x] Was this feature demo'd and the design review approved?

### CODE REVIEW

**Review by ghanlohar — APPROVED** [review]
Looks good to me.

**Review by riginoommen — APPROVED** [review]

**Review by riginoommen — CHANGES_REQUESTED** [review]
For few other native SPAs like user group SPA this update seems missing

**Comment by hybridx** [comment]
> For few other native SPAs like user group SPA this update seems missing

@riginoommen This is specifically for all the angular SPAs

File: packages/common/footer.json [hunk 1]
```diff
@@ -3,12 +3,12 @@
         "category": "Quick Links",
         "links": [
             {
-                "text": "One Platform in Mojo",
-                "href": "https://mojo.redhat.com/groups/pnt-devops/projects/one-portal/"
+                "text": "One Platform in Source",
```
Comment (sayak-sarkar) [inline packages/common/footer.json:6]:
```suggestion
                "text": "One Platform on The Source",
```

**Review by sayak-sarkar — APPROVED** [review]

**Review by riginoommen — APPROVED** [review]

**Review by sayak-sarkar — APPROVED** [review]

**Pull request merged**